tests/dummy/app/templates/map.hbs

```diff
@@ -42,44 +42,44 @@
   </div>
   <div class="ui inernally celled grid">
     <div class="ui very wide right sidebar treeview pushable">
-      {{flexberry-maplayers
-        class="styled"
-        cswConnections=cswConnections
-        leafletMap=leafletMap
-        layers=(get-with-dynamic-actions this "model.rootLayer.layers"
-          hierarchyPropertyName="layers"
-          pathKeyword="layerPath"
-          dynamicActions=(array
-            (hash
-              on="add"
-              actionName="onMapLayerAdd"
-              actionArguments=(array "{% layerPath %}")
-            )
-            (hash
-              on="edit"
-              actionName="onMapLayerEdit"
-              actionArguments=(array "{% layerPath %}")
-            )
-            (hash
-              on="remove"
-              actionName="onMapLayerRemove"
-              actionArguments=(array "{% layerPath %}")
-            )
-            (hash
-              on="changeVisibility"
-              actionName="onMapLayerChangeVisibility"
-              actionArguments=(array "{% layerPath %}.visibility")
+        {{flexberry-maplayers
+          class="styled"
+          cswConnections=cswConnections
+          leafletMap=leafletMap
+          layers=(get-with-dynamic-actions this "model.hierarchy"
+            hierarchyPropertyName="layers"
+            pathKeyword="layerPath"
+            dynamicActions=(array
+              (hash
+                on="add"
+                actionName="onMapLayerAdd"
+                actionArguments=(array "{% layerPath %}")
+              )
+              (hash
+                on="edit"
+                actionName="onMapLayerEdit"
+                actionArguments=(array "{% layerPath %}")
+              )
+              (hash
+                on="remove"
+                actionName="onMapLayerRemove"
+                actionArguments=(array "{% layerPath %}")
+              )
+              (hash
+                on="changeVisibility"
+                actionName="onMapLayerChangeVisibility"
+                actionArguments=(array "{% layerPath %}.visibility")
+              )
             )
           )
-        )
-        add=(action "onMapLayerAdd" "model.rootLayer")
-      }}
-    </div>
+          add=(action "onMapLayerAdd" "model.hierarchy")
+        }}
+      </div>
     <div class="row">
       <div class="sixteen wide column">
         {{#flexberry-maptoolbar
           leafletMap=leafletMap
-          layers=model.rootLayer.layers as |mapToolbar|
+          layers=model.hierarchy as |mapToolbar|
```

Comment:
thought identity and search tools are already set up to work with hierarchical layers, so I decided not to change, cuz model.hierarchy equals model.rootLayer.layers